tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
     "outDir": "./build",
-    "strict": true,
```

Comment:
What was the reason for removing strict checking?

Reply:
This is something that we also had to do with the pollers. In order for AWS IAM auth to work, we need the `mysql2` package, but this package doesn't ship with the correct type defs, causing TS compilation to fail.
```
Could not find a declaration file for module 'mysql2'
```